IdentityServer/v6/docs/content/quickstarts/1_client_credentials.md

```diff
@@ -64,7 +64,7 @@ You can now use your favorite text editor to edit or view the files.
 Add the IdentityServer project to the solution:
 ```console
 cd ..
-dotnet sln add ./src/IdentityServer/IdentityServer.csproj
+dotnet sln add .\src\IdentityServer\IdentityServer.csproj
```

Comment:
IIRC forwards slashes work xplat. Backwards not

Reply:
Oh shoot, that's right. Fixed!